src/components/common/BreadCrumbs.vue

```diff
@@ -0,0 +1,36 @@
+<script>
+import AppButton from '@/components/common/App/AppButton.vue';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  components: { AppButton }
+});
+</script>
+
+<template>
+  <div class="bread-crumbs">
+    <AppButton
+      color-scheme="dark-reverse"
+      arrow="left"
+      class="bread-crumbs__button"
+      @click="this.$router.go(-1)"
```

Comment:
Does this router navigation work? `this` keyword shouldn't be used in a template

Reply:
😄 yes, it works. But without "this" it works equally well 😉

Reply:
Yeah, probably better to remove it 😁 I would expect some console warning about it